src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/components/step_index_pattern/index.js

```diff
@@ -10,6 +10,11 @@ export function renderStepIndexPattern(
   esService,
   goToNextStep,
 ) {
+  const node = document.getElementById(domElementId);
```

Comment:
Maybe make this into a function so both places can use it without repeating code?

Reply:
Sure!